app/main.py

```diff
@@ -1,3 +1,15 @@
 def cache(func):
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def inner(*args):
+        if len(cache_dict) < 1:
+            print("Calculating new result")
+            cache_dict[args] = func(*args)
+        else:
+            if args in cache_dict.keys():
```

Comment:
```suggestion
            if args in cache_dict:
```
More clear way to check key presence